src/components/date-picker/date-picker.scss

```diff
@@ -1,233 +1,13 @@
-@import '../../style/internal/mdc-variables';
-
-@import '@limetech/mdc-theme/mdc-theme';
-@import '../../../node_modules/flatpickr/dist/flatpickr.min.css';
-
-$inactiveTextColor: var(
-    --mdc-theme-text-disabled-on-background,
-    #{$mdc-theme-text-disabled-on-background}
-);
-
-$datepickerHeaderColor: var(--lime-primary-color, #{$mdc-theme-primary});
-$datepickerSelectedColor: $datepickerHeaderColor;
-$datepickerHoverHighlightColor: $datepickerHeaderColor;
-$datepickerTextOnThemeColor: var(
-    --lime-on-primary-color,
-    #{$mdc-theme-on-primary}
-);
-
-.container {
-    display: inline-block;
-    position: relative;
-    width: 100%;
+limel-portal {
+    height: 3.5rem;
+    z-index: -999999;
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    right: 0;
+    left: 0;
```

Comment:
This styling seems a little odd to me:

1. Setting both the `bottom` and `top` values should override the `height` value.
2. Since `position` is `absolute`, this will make the `limel-portal` element the exact same size as the closest positioned parent, which seems a little odd. Is the intention to set the element to the full size of the viewport? If so, `position: fixed` should be used.
3. This is setting the size of the `limel-portal` element, not the size of the `div` located in the root of the `body`, where the content is actually rendered.

I'm guessing this might all be left over from earlier attempts at making flatpickr render correctly?

Reply:
I don't feel to confident in writing css so if you think we should change this I'm all ears. (This change is related to the same as I explained in this comment btw:

> It's because I wanted it to render above the input element and not "over it" when rendering upwards

Reply:
This is the better way to do it, than setting `height` 👌